mods/persistence/modules/science/files/programs/bluprintr.dm

```diff
@@ -159,7 +159,8 @@
 	data["current_protolathe"] = current_protolathe ? current_protolathe : "Not set"
 
 	var/datum/computer_network/network = computer.get_network()
-	data["blueprint"] = uppertext(open_file ? open_file.recipe.name : "Not set")
+	var/datum/fabricator_recipe/open_recipe = open_file?.get_recipe()
+	data["blueprint"] = uppertext(open_file ? open_recipe.name : "Not set")
```

Comment:
This can be inlined to data["blueprint"] = uppertext(open_file?.get_recipe().name || "Not set") if you want